net/README.md

```diff
@@ -32,11 +40,11 @@ NOTE: This example uses GCE.  If you are using AWS EC2, replace `./gce.sh` with
 ```bash
 $ cd net/
 $ ./gce.sh create -n 5 -c 1     #<-- Create a GCE testnet with 5 additional nodes (beyond the bootstrap node) and 1 client (billing starts here)
-$ ./init-metrics.sh $(whoami)   #<-- Recreate a metrics database for the testnet and configure credentials
+$ ./gce.sh info                 #<-- Show the current network node IPs so you can SSH into them if needed
 $ ./net.sh start                #<-- Deploy the network from the local workspace and start processes on all nodes including bench-tps on the client node
 $ ./ssh.sh                      #<-- Show a help to ssh into any testnet node to access logs/etc
 $ ./net.sh stop                 #<-- Stop running processes on all nodes
-$ ./gce.sh delete               #<-- Dispose of the network (billing stops here)
+$ ./gce.sh delete               #<-- Dispose of the network (billing stops here). This can be executed at any point to reset everything into "clean" state
 ```
```

Comment:
*bikeshedding*

I would avoid end line comments.
They cause unnecessary noise in diffs, and are harder to maintain when they get longer.

````suggestion
```bash
cd net/

# Create a GCE testnet with 5 additional nodes (beyond the bootstrap node) and 1
# client (billing starts here).
./gce.sh create -n 5 -c 1

# Show the current network node IPs so you can SSH into them if needed.
./gce.sh info

# Deploy the network from the local workspace and start processes on all nodes
# including bench-tps on the client node.
./net.sh start

# Shows how to ssh into any testnet node to access logs/etc.
./ssh.sh

# Stop running processes on all nodes.
./net.sh stop

# Dispose of the network (billing stops here). This can be executed at any point
# to reset everything into a clean state.
./gce.sh delete
```
````

Reply:
No bikeshedding, your suggestion is good. I am preparing a better version of that entire file anyway.